src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,32 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private List<Bulldozer> bulldozersList = new ArrayList<>();
```

Comment:
you reinitialize this variable every time in the get() method, so you can make it local (declare in the method)

Reply:
done